tests/installation/bootloader_s390.pm

```diff
@@ -260,21 +260,20 @@ sub create_encrypted_part_dasd {
 }
 
 sub format_dasd {
+    # agama bring dasd online through the jsonnet profile
+    return if get_var('AGAMA');
     my $self = shift;
     my $r;
     my $dasd_path = get_var('DASD_PATH', '0.0.0150');
 
     # activate install-shell to do pre-install dasd-format
     select_console('install-shell');
 
-    # agama dasd is already online due it is specified in parmfile
-    unless (get_var('AGAMA')) {
-        # bring dasd online
-        # exit status 0 -> everything ok
-        # exit status 8 -> unformatted but still usable (e.g. from previous testrun)
-        $r = script_run("dasd_configure $dasd_path 1");
-        die "DASD in undefined state" unless (defined($r) && ($r == 0 || $r == 8));
-    }
+    # bring dasd online
+    # exit status 0 -> everything ok
+    # exit status 8 -> unformatted but still usable (e.g. from previous testrun)
+    $r = script_run("dasd_configure $dasd_path 1");
+    die "DASD in undefined state" unless (defined($r) && ($r == 0 || $r == 8));
```

Comment:
we don't want to bring it online during booting, we want to do nothing with the disk.

Reply:
I added `return if get_var('AGAMA');` under `sub format_dasd {`, is it expected? Then it won't bring DASD online, check the device or bring it down.

Reply:
why you need to touch this code?
the existent code does `unless (get_var('AGAMA')) {` so it doesn't run in agama.

Reply:
After removing the `rd.zdev=dasd,0.0.0150` from setting `EXTRABOOTPARAMS` , there's no dasd device when booting. So can not checking the device, for example list the dasd device by command `lsdasd` (line279). So skip the whole function of format_dasd.

Reply:
The VR is pass now, but I keep the [WIP] here, still need to think twice whether the changes in `bootloader_s390.pm`  is suitable. We still have a few testsuite with `rd.zdev=dasd,0.0.0150` in `EXTRABOOTPARAMS` setting in s390x zvm testing, I'll run some VRs for them next Monday.
BTW, do I need to change those testsuites to use `dasd=true` in MR?